asset/cloud-client/quickstart_exportassets_test.py

```diff
@@ -43,8 +44,8 @@ def asset_bucket(storage_client):
 
 
 def test_export_assets(asset_bucket, capsys):
-    dump_file_path = "gs://", asset_bucket, "/assets-dump.txt"
+  dump_file_path = 'gs://{}/assets-dump.txt'.format(asset_bucket)
```

Comment:
Insert two spaces here, looks like your indention was off.  Please also add the `result()` addition to `quickstart_exportassets.py` so I can comment on it as well.